app/controllers/idv/in_person/public/usps_locations_controller.rb

```diff
@@ -3,15 +3,34 @@
 module Idv
   module InPerson
     module Public
+      class UspsLocationsError < StandardError
+        def initialize
+          super('Unsupported characters in address field.')
```

Comment:
no biggie, but I noticed sometimes there's a pattern to add these error messages as constants. Just wanted to pop that in here as food for thought but non-blocking.